jenkins-design-language/src/js/components/Linkify.jsx

```diff
@@ -36,5 +37,8 @@ export const Linkify = (props: Props) => {
     );
 };
 
-Linkify.propTypes = OriginalLinkify.propTypes;
+Linkify.propTypes = {
+    ...OriginalLinkify.propTypes,
+    options: PropTypes.object,
```

Comment:
I have the feeling this change is not related to the underlying PR

Reply:
Because lint was not being run during the build, this lint error was not failing the build. Now that lint is being running, this lint error needs to be fixed in order for the build to pass.

Reply:
Yeah got you but then it should not have been in this PR but another PR extending this one, so this PR would show that CI should fail and the other then merged with this fix.